src/app/app.ts

```diff
@@ -22,6 +23,11 @@ export interface MyWindow extends Window {
 }
 
 
+let appbarObj = new AppBar({
```

Comment:
Define the appbar below global variables

Reply:
Appbar definition has been moved below the global variables